lightsail-instance-launch-scripts/windows_server/enable_win_narrator.bat

```diff
@@ -0,0 +1,9 @@
+<script>
+    net start audiosrv
+
+    reg add "HKCU\Software\Microsoft\Windows NT\CurrentVersion\Accessibility" /t REG_SZ /v Configuration /d narrator /f
+
+    narrator
+</script>
+
+<persist>true</persist>
```

Comment:
What does this do?

Reply:
Ah nevermind! Found [the answer.](https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2-windows-user-data.html)

> By default, user data scripts run one time when you launch the instance. To run the user data scripts every time you reboot or start the instance, add <persist>true</persist> to the user data.

Reply:
This is based on a re:post request where a Visually impaired customer of AWS wanted to enable audio and start narrator ( a build in accessibility feature in windows) the script allows someone to set this up using launch script so that when the log in using an RDP session the narrator start with no intervention there on.

Reply:
in all honestly I shouldnt need the persist true ... as I am editing the reg key ... however I have tested it without persist = true .. and the narrator doesn't start without it as expected. So I have to leave it as true for the accessibility feature to start every time.